web/src/components/EngineControlPanel.vue

```diff
@@ -150,8 +150,8 @@
           >).
         </p>
         <p>
-          If you have it already, please start the Plugin Engine, and connect to
-          it.
+          Once installed, please start the Plugin Engine, and fill the
```

Comment:
..., and enter the connection token below.

Reply:
fixed in another PR, thanks.